modnet/models.py

```diff
@@ -370,127 +397,113 @@ def fit_preset(
         """
 
         if callbacks is None:
-            es = keras.callbacks.EarlyStopping(
+            es = tf.keras.callbacks.EarlyStopping(
                 monitor="loss",
                 min_delta=0.001,
                 patience=100,
                 verbose=verbose,
                 mode="auto",
                 baseline=None,
-                restore_best_weights=True,
+                restore_best_weights=False,
```

Comment:
I know we discussed this, but I thought the solution was to watch the validation loss instead of the overall loss, rather than not restoring the best weights?

Reply:
True, but for a final refit I'm not sure if leaving let's say 10% samples out (only used for validation), is the best option. To be discussed...

Reply:
(We didn't end up discussing this so I will raise an issue)